freqtrade/exchange/binance.py

```diff
@@ -228,3 +249,29 @@ def funding_fee_cutoff(self, open_date: datetime):
         :return: The cutoff open time for when a funding fee is charged
         """
         return open_date.minute > 0 or (open_date.minute == 0 and open_date.second > 15)
+
+    def get_maintenance_ratio_and_amt(
+        self,
+        pair: str,
+        nominal_value: Optional[float] = 0.0,
+    ) -> Tuple[float, Optional[float]]:
+        """
+        Formula: https://www.binance.com/en/support/faq/b3c689c1f50a44cabb3a84e663b81d93
+
+        Maintenance amt = Floor of Position Bracket on Level n *
+          difference between
+              Maintenance Margin Rate on Level n and
+              Maintenance Margin Rate on Level n-1)
+          + Maintenance Amount on Level n-1
+        :return: The maintenance margin ratio and maintenance amount
+        """
+        if nominal_value is None:
```

Comment:
The way this method is currently called (from `freqtradebot.py`)- `nominal_value` will always be 0.0 (the default) as the parameter is never provided.

Reply:
That's not true

![](https://i.imgur.com/i5uZ17p.png)

Reply:
Is this resolved?